### PR TITLE
Short note on ESPOS (BSC#1207062)

### PR DESCRIPTION
**Pull request by cwickert**
### PR creator: Description

Added a note on ESPOS (as per @ajaeger's suggestion)

Additionally, I'd like to clarify what "supported" means because this seems to confuse many people. Supported means not only 'technically possible' or 'clean package upgrades' but also implies that both the source and the target of the migration are actively supported. As support for the source depends on the subscription (general support or ESPOS/LTSS), this depends on the system. I'm thinking of something along the lines of:

> Upgrades paths are officially supported when two conditions are met:
> 1. The upgrade is technically possible. &suse; ensures clean upgrade paths for all SUSE Linux Enterprise packages. They either get updated by newer versions of the same package, or repaced or obsoleted by other packages.
> 2. Both the source and the target of the migration are under active support by SUSE.  This can be general support, Extended Service Pack Overlap Support (ESPOS), or Long Term Service Pack Support (LTSS).
> 

What do you think?

### PR creator: Are there any relevant issues/feature requests?

* [bsc#1207062](https://bugzilla.suse.com/show_bug.cgi?id=1207062)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/sle_update_upgrading.xml [hunk 1]
```diff
@@ -309,8 +309,7 @@
     <term>Upgrading from &productname; 15 SP3</term>
     <listitem>
      <para>
-      Upgrading from &productnameshort;&nbsp;15&nbsp;SP1 or SP2 is supported
-      both online and offline.
+      Upgrading from &productnameshort;&nbsp;15&nbsp;SP3 is supported both online and offline.
```
Comment (ajaeger) [inline xml/sle_update_upgrading.xml:312]: This change looks wrong, let's discuss further.

File: xml/sle_update_upgrading.xml [hunk 2]
```diff
@@ -115,8 +115,8 @@
     The easiest upgrade path is consecutively installing all service packs. For
     the &sle;&nbsp;&product-ga; product line (GA and the subsequent service
     packs) it is also supported to skip up to two service packs when upgrading.
-    For example, upgrading from &slea;&nbsp;15&nbsp;GA to 15&nbsp;SP3 is
-    supported (as long as &slea;&nbsp;15&nbsp;GA is supported).
+    For example, upgrading from &slea;&nbsp;15&nbsp;SP1 to 15&nbsp;SP4 is
+    supported (as long as &slea;&nbsp;15&nbsp;SP1 is supported).
```
Comment (ajaeger) [inline xml/sle_update_upgrading.xml:119]: This change looks wrong, let's discuss per email.

File: xml/sle_update_upgrading.xml [hunk 3]
```diff
@@ -372,6 +371,20 @@
...
+    Support</productname> (ESPOS) under the same conditions as LTSS. Thus,
+    wherever the list above speaks about LTSS, products with ESPOS are included
+    as well.
+   </para>
+   <para>
+    For more information about ESPOS, refer to the documentation of the respective &sle; product
+    and the <link xlink:href="https://www.suse.com/support/policy-products/">Product Lifecycle
+     Support Policies</link> webpage.
+   </para>
+  </note>
```
Comment (ajaeger) [inline xml/sle_update_upgrading.xml:387]: I'm fine with this change.

**Review by ajaeger — COMMENTED** [review]
This needs further discussion before I can approve or request changes.

**Comment by ajaeger** [comment]
I see in the intermediate commits some additions that are not in the summary "1 file changed" - could you build a guide with complete chapter, please?

File: xml/sle_update_upgrading.xml (hunk 1 above)
Comment (cwickert) [reply xml/sle_update_upgrading.xml:312]: Simple upgrades have always been supported both offline and online. Why would that change?

Comment (ajaeger) [reply xml/sle_update_upgrading.xml:312]: Now reading it in context: This is fine.

File: xml/sle_update_upgrading.xml (hunk 2 above)
Comment (ajaeger) [reply xml/sle_update_upgrading.xml:119]: Yes, this is fine.

**Review by ajaeger — APPROVED** [review]
LGTM, thanks. The extra change (GA -> SP1) that is not mentioned in commit message confused me initially, reading it now in full context, the change is fine as well for SP4.

The ESPOS addition should go to older and newer guides as well - please ensure it's in SP1, SP2, SP3 and SP5!

Thanks!

**Comment by cwickert** [comment]
Backported to all branches down to SLE 15 SP1.

**Pull request merged**